api/src/main/java/com/github/retrooper/packetevents/util/PacketTransformationUtil.java

```diff
@@ -48,7 +50,7 @@ public static PacketWrapper<?>[] transform(PacketWrapper<?> wrapper) {
                 PacketWrapper<?>[] output = new PacketWrapper[len];
                 for (int i = 0; i < len; i++) {
                     Equipment equipment = entityEquipment.getEquipment().get(i);
-                    output[i] = new WrapperPlayServerEntityEquipment(entityEquipment.getEntityId(), equipment);
+                    output[i] = new WrapperPlayServerEntityEquipment(entityEquipment.getEntityId(), Collections.singletonList(equipment));
```

Comment:
Does WrapperPlayServerEntityEquipment expose the passed collection somehow or what would the benefit of making this a singleton be here?

Reply:
Because of old code